apps/core/scenes/search/Search.js

```diff
@@ -81,21 +82,20 @@ class Search extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
 
-    const defaultDepartureDate = DateTime.local().toISODate();
-    const defaultReturnDate = DateTime.local()
-      .plus({ days: 2 })
-      .toISODate();
+    const defaultDepartureDate = new Date();
+    const defaultReturnDate = DateFNS.addDays(defaultDepartureDate, 2);
```

Comment:
If you look in the PR where I added Date and Currency Formatters, there is a similar function:
https://github.com/kiwicom/margarita/pull/117/files#diff-cbd47da69efb4e89e3d7340c962d5241

Reply:
Thanks, I've considered it and my problem is that it also converts date to UTC which is not that ideal, because currently we use local timezone in whole search form.
But maybe it will make sense to switch to UTC internally and just render localised strings for date buttons.
Next thing is that if we don't want to get rid of `date-fns` completely and replace it with our custom `DateUtils/Formatter` implementation I don't see much of the added value in duplicating what is already part of the `date-fns`.

Reply:
@tbergq What was the reasoning behind having a custom DateFormatter instead of using `date-fns` in `mobile`?

Reply:
The reason was that we was using `moment`, but moment was not good enough. 

The best we found for solving our problems was to use Javascript `Intl` package. If `date-fns` solves the same, we could always replace it in `mobile` in the future.

Reply:
Thanks for explanation. I don't think we are currently facing any limitations with `date-fns`.
But if you think we can expect same problems or limitations as in `mobile` later, I can definitely switch to our custom solution, but I'll probably need to extend it to also handle non-utc dates.